### ISSUE TITLE
[ENHANCEMENT] Do not expose port `6060` with NDM

### ISSUE THREAD

**Issue by Vicente-Cheng** [harvester/harvester#4212]
**Is your enhancement related to a problem? Please describe.**
The port `6060` will be occupied by NDM due to the `hostNetwork` parameter.
We may try to limit it.

**Describe the solution you'd like**
TBD

**Describe alternatives you've considered**

**Additional context**

**Comment by Vicente-Cheng** [comment]
wrong issue, due to harvester-node-manager exposing hostNetwork

So port 6060 would be occupied by NDM, we could try to limit the hostNetowk.

**Comment by Vicente-Cheng** [comment]
It needs too much effort to listen to event w/o `hostNetwork: true`; we choose to disable profiler by default.

If we have another reason to create NDM w/o `hostNetwork: true`, we could do more discuss for it.

**Comment by TachunLin** [comment]
Verified **fixed** on `master-d3dc81ca-head` (23/01/04). Close this issue.

### Result
- [x]  The Harvester host and  the node-disk manager does not listen to port :6060 

    <details><summary>Test Result $~~$ $\color{green}{\textsf{PASS}}$ </summary>
    On v1.3 master, **no port** `6060` is listening on the host
    
    ```    
    node1:~ # ss -tnal | grep 6060
    
    node1:~ # sudo lsof -i -P -n | grep LISTEN
    sshd       2147    root    3u  IPv4  23021      0t0  TCP *:22 (LISTEN)
    sshd       2147    root    4u  IPv6  23023      0t0  TCP *:22 (LISTEN)
    etcd       2740    root    7u  IPv4  25596      0t0  TCP 127.0.0.1:2380 (LISTEN)
    etcd       2740    root    8u  IPv4  25600      0t0  TCP 192.168.50.101:2380 (LISTEN)
    etcd       2740    root    9u  IPv4  29697      0t0  TCP 192.168.50.101:2379 (LISTEN)
    etcd       2740    root   10u  IPv4  29698      0t0  TCP 127.0.0.1:2382 (LISTEN)
    etcd       2740    root   11u  IPv4  29699      0t0  TCP 127.0.0.1:2379 (LISTEN)
    etcd       2740    root   18u  IPv4  26975      0t0  TCP 127.0.0.1:2381 (LISTEN)
    kube-apis  2865    root    7u  IPv6  30735      0t0  TCP *:6443 (LISTEN)
    cloud-con  3131    root    3u  IPv4  27205      0t0  TCP 127.0.0.1:10258 (LISTEN)
    kube-prox  3217    root    7u  IPv4  23520      0t0  TCP 127.0.0.1:10256 (LISTEN)
    kube-prox  3217    root    8u  IPv4  27254      0t0  TCP 127.0.0.1:10249 (LISTEN)
    calico-no  5712    root    9u  IPv4  30621      0t0  TCP 127.0.0.1:9099 (LISTEN)
    calico-no  5712    root   14u  IPv6  38075      0t0  TCP *:9091 (LISTEN)
    rke2      18403    root    8u  IPv6  79660      0t0  TCP *:9345 (LISTEN)
    container 18470    root  220u  IPv4  89113      0t0  TCP 127.0.0.1:10010 (LISTEN)
    kubelet   18755    root   17u  IPv4  80733      0t0  TCP 127.0.0.1:10248 (LISTEN)
    kubelet   18755    root   20u  IPv6  83534      0t0  TCP *:10250 (LISTEN)
    kube-cont 18985    root    7u  IPv4  89273      0t0  TCP 127.0.0.1:10257 (LISTEN)
    kube-sche 19035    root    7u  IPv4  84934      0t0  TCP 127.0.0.1:10259 (LISTEN)
    kube-vip  22154    root    7u  IPv6  98650      0t0  TCP *:2112 (LISTEN)
    
    ```

   Compared with v1.2.1, we can found the port 6060 is listening on the host 
   
    ```
    n1-121:~ # ss -tnal | grep 6060
    LISTEN 0      4096                *:6060             *:*          
    n1-121:~ # sudo lsof -i -P -n | grep LISTEN
    sshd       1921    root    3u  IPv4  24094      0t0  TCP *:22 (LISTEN)
    sshd       1921    root    4u  IPv6  24096      0t0  TCP *:22 (LISTEN)
    etcd       2302    root    7u  IPv4  24244      0t0  TCP 127.0.0.1:2380 (LISTEN)
    etcd       2302    root    8u  IPv4  24248      0t0  TCP 192.168.122.27:2380 (LISTEN)
    etcd       2302    root    9u  IPv4  24249      0t0  TCP 127.0.0.1:2379 (LISTEN)
    etcd       2302    root   10u  IPv4  24250      0t0  TCP 192.168.122.27:2379 (LISTEN)
    etcd       2302    root   14u  IPv4  16035      0t0  TCP 127.0.0.1:2381 (LISTEN)
    kube-apis  2424    root    7u  IPv6  24308      0t0  TCP *:6443 (LISTEN)
    cloud-con  2692    root    3u  IPv4  25440      0t0  TCP 127.0.0.1:10258 (LISTEN)
    kube-prox  2869    root    7u  IPv4  27977      0t0  TCP 127.0.0.1:10249 (LISTEN)
    kube-prox  2869    root   11u  IPv4  24472      0t0  TCP 127.0.0.1:10256 (LISTEN)
    calico-no  4673    root    9u  IPv4  31010      0t0  TCP 127.0.0.1:9099 (LISTEN)
    calico-no  4673    root   13u  IPv6  34946      0t0  TCP *:9091 (LISTEN)
    rke2      15333    root    8u  IPv6  75350      0t0  TCP *:9345 (LISTEN)
    container 15380    root  227u  IPv4  71501      0t0  TCP 127.0.0.1:10010 (LISTEN)
    kubelet   15431    root   31u  IPv4  74612      0t0  TCP 127.0.0.1:10248 (LISTEN)
    kubelet   15431    root   32u  IPv6  75449      0t0  TCP *:10250 (LISTEN)
    kube-sche 15638    root    7u  IPv4  70534      0t0  TCP 127.0.0.1:10259 (LISTEN)
    kube-cont 15661    root    7u  IPv4  71595      0t0  TCP 127.0.0.1:10257 (LISTEN)
    node-disk 21703    root    3u  IPv6 100929      0t0  TCP *:6060 (LISTEN)
    kube-vip  31591    root    7u  IPv6 192083      0t0  TCP *:2112 (LISTEN)
    
    ```


    </details>


### Test Information
* Test Environment: 3 nodes harvester on baremetal machines
* Harvester version: `master-d3dc81ca-head` (23/01/04)

### Verify Steps
<details><summary>Execution Steps</summary>

  1. Prepare 3 nodes Harvester cluster
  2. Execute the following command on node

    ```
    ss -tnal | grep 6060
    
    sudo lsof -i -P -n | grep LISTEN
    ```

</details>
